### ISSUE TITLE
How to improve prediction?

### ISSUE THREAD

**Issue by abdou31** [experiencor/keras-yolo3#278]
Hello,
Thanks for this project, it help me to predict face ( if wear mask or not )  
but I got some problems ( drawing two boxes of differents class on the same object ): 

![image](https://user-images.githubusercontent.com/19480228/83898959-ef7a8980-a757-11ea-90cc-3941e6ee9b85.png)

How can I improve the prediction?

**Comment by h33p** [comment]
It appears that the model needs more training data.

Otherwise, I would calculate the overlap between 2 boxes and if they are roughly the same, discard the one with lower confidence.

**Comment by abdou31** [comment]
Good idea, but do you have a piece of code of this idea???
